docs/running-minio-prometheus-exporter.md

```diff
@@ -0,0 +1,57 @@
+# Running Minio Prometheus Exporter [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io)
+
+This recipe explains how to run a Minio Prometheus exporter.
+
+
+## 1. Prerequisites
+
+* Install Minio Server from [here](https://docs.minio.io/docs/minio-quickstart-guide).
+* Get the minio-exporter from [here](https://github.com/joe-pll/minio-exporter).
+
+## 2. Running the exporter
+
+Once the Minio server starter there are two ways to run the exporter, via building the binaries or using the docker image.
+
+### Run the exporter using the binaries
+```bash
+make
+./minio_exporter [flags]
+```
+
+| Flag | Description | Default |
+| ---- | ------------| ------- |
+| version | Print version number and leave | |
+| web.listen-address | The address to listen on to expose metrics. | *:9290* |
+| web.telemetry-path | The listening path for metrics. | */metrics* |
+| minio.server | The URL of the minio server. Use HTTPS if Minio accepts secure connections only. | *http://localhost:9000* |
+| minio.access-key | The value of the Minio access key. It is required in order to connect to the server | "" |
+| minio.access-secret | The calue of the Minio access secret. It is required in order to connect to the server | "" |
+| minio.bucket-stats | Collect statistics about the buckets and files in buckets. It requires more computation, use it carefully in case of large buckets. | false |
+
+```bash
+./minio_exporter -minio.server minio-host.example:9000 -minio.access-key "login_name" -minio.access-secret "login_password"
+```
+
+### Running the exporter using docker
+
+```bash
+docker pull joepll/minio-exporter
+docker run -p 9290:9290 joepll/minio-exporter -minio.server "minio.host:9000" -minio.access-key "login_name" -minio.access-secret "login_secret"
+```
+
+The same result can be achieved with Enviroment variables.
+* **LISTEN_ADDRESS**: is the exporter address, as the option *web.listen-address*
+* **METRIC_PATH**: the telemetry path. It corresponds to *web.telemetry-path*
+* **MINIO_URL**: the URL of the Minio server, as *minio.server*
+* **MINIO_ACCESS_KEY**: the Minio access key (*minio.access-key*)
+* **MINIO_ACCESS_SECRET**: the Minio access secret (*minio.access-secret*)
+
+
+```bash
+docker run \
+       -p 9290:9290 \
+       -e "MINIO_URL=http://host.local:9000" \
+       -e "MINIO_ACCESS_KEY=loginname" \
+       -e "MINIO_ACCESS_SECRET=password" \
+       joepll/minio-exporter
+```
```

Comment:
Probably add a last step where you can point people to some Prometheus Getting Started resources?